docs/_sources/admin/hwrequirements.md.txt

```diff
@@ -1,62 +1,83 @@
-as little as 32GB of RAM, we recommend a minimum of 64GB. If you have 2 processors, you should double that to 128GB.  The more memory you can install,
-the better the performance you will have.  For most cases, 128GB will be more than enough.
 
-## Disk Space
-We recommend using 2 disks for the operating system, configured as mirror devices.  For best results, use fast disks such as SSDs or NVME for
-the operating system.  These can be as small as 128GB. If you want to speed up DL results, increase this to 256GB and place the squid cache on this device.
+The ability of the Data Library to process large datasets depends on the amount
+of memory installed on the system. While a Data Library may work with
+as little as 32GB of RAM, we recommend 64GB. If you have 2 processors, you
+should double that to 128GB. The more memory you can install,
+the better the performance you will have. For most cases, 128GB will be more
+than enough.
 
-You will need space for your Datasets.  We recommend getting multiple disks so you can mirror the drive and an external disk to backup the data regularly.
-Mirroring is not considered a backup, it is simply a convenience for higher availability.  A separate backup of your data is very important. The amount of disk
-space you may need is unique to your installation.  We suggest getting disks that are about twice the size you think you need.  You will use 2 disks to mirror
-the actual data and one disk, preferably an external USB disk, to backup your data.
+## Storage
+
+You _can_ put everything on one disk if necessary, but we recommend mirroring
+across 2 disks in case of a single disk failure. For best results,
+use fast disks such as SSDs or NVME for the operating system. These can be as
+small as 128GB for the OS. If you want to speed up DL results, increase this to
+256GB
+and place the squid cache on this device. You should try to get Enterprise hard
+disk drives. They are more generally reliable and last significantly longer.
+
+You will also need space for your data. If you're not installing on the OS disk,
+we recommend getting multiple disks so you can mirror the drive in case of a
+single disk failure.
+Mirroring is not considered a backup, so you should add an external disk as a
+regular backup. The amount of disk
+space you may need is unique to your installation. We suggest getting disks that
+are about twice the size you think you need.
 
 ## Networking
-You should use the highest network speed you have access to.  If you have a 10Gb network, you should try to place your server on this network, with a 10Gb
-network card on your server.  But this is entirely optional.  Whatever network you have access to will work.
+
+You should use the highest network speed you have access to. If you have a 10Gb
+network, you should try to place your server on this network, with a 10Gb
+network card on your server. But this is entirely optional. Whatever network you
+have access to will work.
```

Comment:
What's the thinking behind this recommendation? If storage and compute are on separate nodes then it's important for the network between them to be fast, but the DL doesn't use high bandwidth to the internet.

Reply:
My thought is if they have a 10Gb network connection they should use it.  But you're right, it's not very important, but it would improve page load times internally, no?.  This wording hasn't changed from the previous version though.

Reply:
I understand that you're not changing it here, and this conversation doesn't have to be resolved now. Just something I noticed while reviewing.

Disk I/O is almost always the bottleneck. Not much data goes over the network--that's the beauty of Ingrid.